tests/geo/test_find_in_string.py

```diff
@@ -0,0 +1,481 @@
+        """
+        test_samples = [
+            (
+                "Fur Museum, 7884 Fur, Denmark.",
+                [
+                    Match(
+                        substring_range=(22, 29),
+                        substring="Denmark",
+                        normalized="Denmark",
+                        score=1.0,
+                    )
+                ],
+            ),
+            (
+                "The University of Queensland Diamantina Institute, The University of Queensland, Translational "
+                "Research Institute, Woolloongabba, Queensland, Australia",
+                [
+                    Match(
+                        substring_range=(142, 151),
+                        substring="Australia",
+                        normalized="Australia",
+                        score=1.0,
+                    )
+                ],
+            ),
+            (
+                "Institute of Plant Sciences, University of Bern, Bern 3005, Switzerl.",
+                [
+                    Match(
+                        substring_range=(60, 68),
+                        substring="Switzerl",
+                        normalized="Switzerland",
+                        score=0.8421052631578947,
+                    )
+                ],
+            ),
+            # check blacklist and whitelist
+            (
+                "0000 0004 0581 2008, grid. 451052. 7, Essex Partnership University NHS Foundation Trust, Essex SS11 "
+                "7XX UK",
+                [
+                    Match(
+                        substring_range=(104, 106),
+                        substring="UK",
+                        normalized="United Kingdom",
+                        score=1.0,
+                    )
+                ],
+            ),
+            (
+                "University of Bern Hochschulstrasse 4 3012CH Bern Switzerland.",
+                [
+                    Match(
+                        substring_range=(50, 61),
+                        substring="Switzerland",
+                        normalized="Switzerland",
+                        score=1.0,
+                    )
+                ],
+            ),
+            (
+                "Divine Word University, Konedobu, NCD 131, Madang, Papua New Guinea.",
+                [
+                    Match(
+                        substring_range=(51, 67),
+                        substring="Papua New Guinea",
+                        normalized="Papua New Guinea",
+                        score=1.0,
+                    )
+                ],
+            ),
+        ]
+        for original, expected_match in test_samples:
+            with self.subTest(original=original, expected_match=expected_match):
+                extracted_country = find_country_in_string(original)
+                self.assertEqual(extracted_country, expected_match)
+
+    def test_find_multiple_matches_country_name_in_string(self):
```

Comment:
Just adding this one again, in case you miss it:
Can you add a test case where there are two different countries extracted from one string?

You have to remember this package is general purpose. Even if it doesn't occur in our dataset, it is valid input and text-scrubber can deal with it. So, let's test it